frontend/app/src/components/import/FileUpload.vue

```diff
@@ -70,11 +76,32 @@ function onSelect(event: Event) {
     selected(target.files[0]);
 }
 
+function clearTimeoutHandler(timeout: Ref<NodeJS.Timeout | null>) {
+  const timeoutVal = get(timeout);
+  if (timeoutVal) {
+    clearTimeout(timeoutVal);
+    set(timeout, null);
+  }
+}
+
+const errorTimeout: Ref<NodeJS.Timeout | null> = ref(null);
+const uploadedTimeout: Ref<NodeJS.Timeout | null> = ref(null);
```

Comment:
```suggestion
const errorTimeout = ref<NodeJS.Timeout>();
const uploadedTimeout = ref<NodeJS.Timeout>();
```

Rather than using explicit `null` let's use undefined as default.
For most of the places now that we removed vuetify and we don't have vue 2.16 we can use `undefined` and keep things simpler.